test/e2e/run.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash -e
+
+export NOTATION_E2E_BINARY_PATH=$(realpath $1)
+if [ ! -f "$NOTATION_E2E_BINARY_PATH" ];then
+    echo "run.sh <notation-binary-path>"
+    exit 1
+fi
+
+# install dependency
+go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo
```

Comment:
Do we need to use a specific version instead of always the latest?

Reply:
Updated to use v2.3.0